kaufui.css

```diff
@@ -16,7 +16,7 @@
 .kaufui .text-thin {
     font-family: "Lato", sans-serif;
     font-weight: 100;
-    font-style: thin;
+    font-style: normal;
```

Comment:
take a look here:
http://www.latofonts.com/lato-free-fonts/ 

there is a thin style, and according to the design document it has to be a thin font